ddtrace/opentelemetry/tracer_provider.go

```diff
@@ -0,0 +1,74 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
```

Comment:
Let's add some package level comments in this file like we do for opentracer (ddtrace/opentracer/tracer.go)

Reply:
Gotcha